app/controllers/ncr/work_orders_controller.rb

```diff
@@ -0,0 +1,99 @@
+module Ncr
+  class WorkOrdersController < ApplicationController
+    before_filter :authenticate_user!
+    before_filter :redirect_if_cart_cant_be_edited, only: [:edit, :update]
+
+    def new
+      @work_order = Ncr::WorkOrder.new
+      @approver_email = self.suggested_approver_email
+      @description = ""
+      render 'form'
+    end
+
+    def errors
+      # @TODO we can use a nested model once we get rid of the Cart requirement
+      errors = []
+      if !@description.is_a?(String) || @description.strip().empty?
```

Comment:
Check out the [`blank?`](http://api.rubyonrails.org/classes/Object.html#method-i-blank-3F) method.